apps/backend/src/modules/users/user.service.ts

```diff
@@ -33,8 +35,12 @@ class UserService implements Service {
 		return Promise.resolve(true);
 	}
 
-	public find(): ReturnType<Service["find"]> {
-		return Promise.resolve(null);
+	public async find(
+		id: number,
+	): Promise<null | ReturnType<UserEntity["toNewObject"]>> {
```

Comment:
`find` should not return `toNewObject` ReturnType. We should have one common type `UserAuthResponseDto` and return it